core/node/rpc/sync/client/local.go

```diff
@@ -100,7 +100,11 @@ func (s *localSyncer) OnUpdate(r *StreamAndCookie) {
 	default:
 		log := dlog.FromCtx(s.syncStreamCtx)
 		log.Error("Cancel client sync operation - client buffer full", "syncId", s.globalSyncOpID)
-		s.cancelGlobalSyncOp()
+
+		err := RiverError(Err_BUFFER_FULL, "Client sync subscription message channel is full").
```

Comment:
suggestion: not to duplicate log and error with same data, it's possible to call .Log() on the error itself to log it.